docs/advanced-usage.rst

```diff
@@ -334,7 +334,8 @@ Brotli Encoding
 
 Brotli is a compression algorithm created by Google with better compression
 than gzip and deflate and is supported by urllib3 if the
-`brotlipy <https://github.com/python-hyper/brotlipy>`_ package is installed.
+`Brotli <https://pypi.org/Brotli>`_ package or
+`brotlicffi <https://github.com/python-hyper/brotlicffi>`_ package is installed.
```

Comment:
Should this read "are installed"? Genuinely not sure as non-native English speaker

Reply:
I think the current phrasing works, or could change packages to plural and then "are installed" would work. Either way. :)